### PR TITLE
Update hot-topics & models, detail windows release in usage

### PR DESCRIPTION
**Pull request by BarfingLemurs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 Inference of [LLaMA](https://arxiv.org/abs/2302.13971) model in pure C/C++
 
 ### Hot topics
-
+- ‼️ BPE tokenizer update: existing Falcon 180B `.gguf` models will need to be reconverted
```
Comment (cebtenzzre) [inline README.md:13]: The update actually breaks all Falcon and Starcoder models, not just Falcon 180B.

File: README.md [hunk 2]
```diff
@@ -89,15 +89,17 @@ as the main playground for developing new features for the [ggml](https://github
 - [X] [Vicuna](https://github.com/ggerganov/llama.cpp/discussions/643#discussioncomment-5533894)
 - [X] [Koala](https://bair.berkeley.edu/blog/2023/04/03/koala/)
 - [X] [OpenBuddy 🐶 (Multilingual)](https://github.com/OpenBuddy/OpenBuddy)
-- [X] [Pygmalion 7B / Metharme 7B](#using-pygmalion-7b--metharme-7b)
+- [X] [Pygmalion/Metharme](#using-pygmalion-7b--metharme-7b)
 - [X] [WizardLM](https://github.com/nlpxucan/WizardLM)
-- [X] [Baichuan-7B](https://huggingface.co/baichuan-inc/baichuan-7B) and its derivations (such as [baichuan-7b-sft](https://huggingface.co/hiyouga/baichuan-7b-sft))
-- [X] [Aquila-7B](https://huggingface.co/BAAI/Aquila-7B) / [AquilaChat-7B](https://huggingface.co/BAAI/AquilaChat-7B)
+- [X] [[Baichuan 1&2](https://huggingface.co/models?search=baichuan-inc/Baichuan) + [derivations](https://huggingface.co/hiyouga/baichuan-7b-sft)
```
Comment (cebtenzzre) [inline README.md:94]: Extra square bracket?

File: README.md [hunk 3]
```diff
@@ -89,15 +89,17 @@ as the main playground for developing new features for the [ggml](https://github
...
+- [X] [Aquila 1&2](https://huggingface.co/models?search=BAAI/Aquila)
 - [X] [Starcoder models](https://github.com/ggerganov/llama.cpp/pull/3187)
 - [X] [Mistral AI v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1)
 - [X] [Refact](https://huggingface.co/smallcloudai/Refact-1_6B-fim)
-- [X] [Bloom](https://github.com/ggerganov/llama.cpp/pull/3553)
+- [X] [Persimmon 8B](https://github.com/ggerganov/llama.cpp/pull/3410)
 - [X] [MPT](https://github.com/ggerganov/llama.cpp/pull/3417)
+- [X] [Bloom](https://github.com/ggerganov/llama.cpp/pull/3553)
```
Comment (cebtenzzre) [inline README.md:101]: I'm not sure what the point of reordering these is.

File: README.md [hunk 4]
```diff
@@ -206,7 +208,17 @@ https://user-images.githubusercontent.com/1991296/224442907-7693d4be-acaa-4e01-8
 
 ## Usage
 
-Here are the steps for the LLaMA-7B model.
+**Quickstart:** You will find prebuilt Windows binaries on the release page.
```
Comment (cebtenzzre) [inline README.md:211]: If we are going to add this it should be a proper section, e.g. `### Quickstart`.

File: README.md (hunk 3 above)
Comment (BarfingLemurs) [reply README.md:101]: I just wanted this list to reflect the order in which these models were added.

File: README.md [hunk 5]
```diff
@@ -206,7 +208,20 @@ https://user-images.githubusercontent.com/1991296/224442907-7693d4be-acaa-4e01-8
...
+You will find prebuilt Windows binaries on the release page.
+
+Simply download and extract the latest zip package of choice: (e.g. `llama-b1380-bin-win-avx2-x64.zip`)
+
+From the unzipped folder, open a terminal/cmd window here and place a pre-converted `.gguf` model file. Test out the main example like so:
+
+```
+.\main -m llama-2-7b.Q4_0.gguf -n 128
+```
+
```
Comment (ggerganov) [inline README.md:221]: These can go in a "Running on Windows with prebuilt binaries" section further down the README.
The "Usage" is oriented primarily towards developers and I prefer to not start with Windows specifics

**Review by ggerganov — APPROVED** [review]

**Pull request merged**